aix/generate_wazuh_packages.sh

```diff
@@ -223,9 +223,14 @@ main() {
         "-b"|"--branch")
           if [ -n "$2" ]
           then
-              wazuh_branch="$2"
-              build_rpm="yes"
-              shift 2
+              if [[ `curl https://api.github.com/repos/wazuh/wazuh-api/branches` =~ "$2" ]] || [[ `curl https://api.github.com/repos/wazuh/wazuh-api/tags` =~ "$2" ]]; then
```

Comment:
This check isn't necessary. `git` automatically checks this while cloning the branch.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/255/commits/ac22541a388ac6ed3a45b864fe89d04baccd4d20